pkg/k8shandler/kibana/consolelink.go

```diff
@@ -0,0 +1,49 @@
+package kibana
+
+import (
+	consolev1 "github.com/openshift/api/console/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewConsoleLink(name, href string) *consolev1.ConsoleLink {
+	return &consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			Location: consolev1.ApplicationMenu,
+			Link: consolev1.Link{
+				Text: "Logging",
+				Href: href,
+			},
+			ApplicationMenu: &consolev1.ApplicationMenuSpec{
+				Section: "Monitoring",
+			},
+		},
+	}
+}
+
+func isConsoleLinkDifferent(current, desired *consolev1.ConsoleLink) (*consolev1.ConsoleLink, bool) {
```

Comment:
It is an odd thing to do for an API, returning to the caller something he already knows. Why is the function's signature not simply ```func isConsoleLinkDifferent(current, desired *consolev1.ConsoleLink) bool```?